src/en/credentials.md

```diff
@@ -0,0 +1,134 @@
+Title: Juju credentials
+
+# Cloud credentials
+
+In order to access your cloud (with the exception of [LXD][lxd]), Juju will
+need to know how to authenticate itself. We use the term “credential” to 
+describe the tokens or keys or secrets needed to spin up resources on clouds.
```

Comment:
This is how I would have written this (terminology also in accordance with command help text):

"Juju needs to import credentials (with the exception of LXD) before it can authenticate to a cloud. A set of credentials (username, password, keys, etc) is represented by a *credential name* that is used to refer to those credentials in subsequent commands."

I don't see the need to say in this document what Juju does ("spin up" stuff) when it talks to a cloud.